### DISCUSSION TITLE
Modding in an Online IDE

### DISCUSSION THREAD

**Discussion by Whop42** [FabricMC/fabric-api#1324] [Mod Dev Support] [answered]
Is there any way to set up a fabric environment in eclipse che, gitpod, or similar? I don't need it to run, but it would help if there were a way to build it from there. I often don't have access to an IDE (school) so this would be very helpful.

**Comment by Whop42** [comment, marked as answer]
Wait never mind, it works with the example mod in gitpod

**Reply by mounderfod** [reply]
That's interesting; does the game run?

**Reply by YTG1234** [reply]
> That's interesting; does the game run?

You need a display server for the game client to run. Although running a server should work just fine.  
Can someone try running the FAPI auto test servers?

**Reply by Whop42** [reply]
I can't figure out a way to get the game to run, but it builds fine.

> Can someone try running the FAPI auto test servers?

What are the FAPI test servers?
